setup.cfg

```diff
@@ -1,7 +1,7 @@
 [flake8]
 max-line-length = 110
 max-doc-length = 79
-ignore = E133, E226, E228, N802, N803, N806, N812, N813, N815, N816, W503
+ignore = E133, E226, E228, N802, N803, N806, N812, N813, N815, N816, W503, E203
```

Comment:
Please insert in alphabetical order (like the rest of them) so it's easier to find.

Similarly for flake8-ignore below.

Reply:
I hadn't noticed it was sorted alphabetically. Done.